dvc/command/version.py

```diff
@@ -34,7 +36,13 @@ def run(self):
         ]
 
         try:
-            repo = Repo()
+            try:
+                repo = Repo()
+            except SCMError:
+                repo = Repo(no_scm=True)
```

Comment:
It is hacky to introduce an option for an affectively broken dvc repo. Maybe let's just catch `SCMError` as we do `NotDvcRepoError` and show what we can?

Reply:
Fixed, thanks. Looks like this can be done without introducing new options (with reductions in confidence, though). :slightly_smiling_face: